src/stories/components/custom-table/row-item/item-core-unit.stories.tsx

```diff
@@ -0,0 +1,144 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import { columns, CoreUnit } from '@ses/core/utils/test.utils';
+import ItemCoreUnit from './item-core-unit';
+import type { ComponentMeta } from '@storybook/react';
+
+export default {
+  title: 'Components/CUTable/ItemCoreUnit',
+  component: ItemCoreUnit,
+  parameters: {
+    chromatic: {
+      viewports: [375, 834, 1194],
```

Comment:
you should include 1440 and 1920 too as this component has has a different look and feel in those breakpoints

Reply:
you are right, it I forgot to put there.
done